index.html

```diff
@@ -1,83 +1,88 @@
-        shortName: "trace-context",
-        format: "markdown",
-        subjectPrefix: "trace-context",
-        wg: "Distributed Tracing Working Group",
-        wgPublicList: "public-trace-context",
-        wgURI: "https://www.w3.org/2018/distributed-tracing/",
-        //isPreview: true, //this will add ugly box saying it's preview
-        otherLinks: [{
-          key: 'Discussions',
-          data: [{
-            value: 'We are on Gitter.',
-            href: 'https://gitter.im/TraceContext/Lobby'
-          }]
-        }],
-        wgPatentURI: "https://www.w3.org/2004/01/pp-impl/108594/status"
-      };
-    </script>
-  </head>
-  <body>
-        <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
-        <section id='sotd' data-include="spec/02-sotd.md" data-include-format='markdown'></section>
 
-        <section data-include="spec/10-overview.md" data-include-format='markdown'></section>
-        <section data-include="spec/11-fields-semantics.md" data-include-format='markdown'></section>
-        
-        <section data-include="spec/20-http_header_format.md" data-include-format='markdown'></section>
+<head>
+  <meta charset='utf-8'>
+  <title>Trace Context</title>
+  <script src='https://www.w3.org/Tools/respec/respec-w3c-common' class='remove' defer></script>
+  <script class='remove'>
+    var respecConfig = {
+      specStatus: "ED",
+      editors: [{
+        name: "Sergey Kanzhelev",
+        company: "Microsoft",
+        companyURL: "https://microsoft.com",
+        w3cid: "103966"
+      },
+      {
+        name: "Morgan McLean",
+        company: "Google",
+        companyURL: "https://google.com",
+        w3cid: "104128"
+      },
+      {
+        name: "Alois Reitbauer",
+        company: "Dynatrace",
+        companyURL: "https://dynatrace.com",
+        w3cid: "48276"
+      }],
+      github: {
+        repoURL: "https://github.com/w3c/trace-context/",
+        branch: "master",
+      },
+      edDraftURI: "https://w3c.github.io/trace-context/",
+      shortName: "trace-context",
+      format: "markdown",
+      subjectPrefix: "trace-context",
+      wg: "Distributed Tracing Working Group",
+      wgPublicList: "public-trace-context",
+      wgURI: "https://www.w3.org/2018/distributed-tracing/",
+      isPreview: true, //this will add ugly box saying it's preview
+      otherLinks: [{
+        key: 'Discussions',
+        data: [{
+          value: 'We are on Gitter.',
+          href: 'https://gitter.im/TraceContext/Lobby'
+        }]
+      }],
+      wgPatentURI: "https://www.w3.org/2004/01/pp-impl/108594/status"
+    };
+  </script>
+</head>
 
-        <section data-include="spec/40-other-protocols.md" data-include-format='markdown'></section>
+<body>
+  <section id='sotd' data-include="spec/02-sotd.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
```

Comment:
I'm very curious of this change. My understanding always was that spec has short abstract, than status and than longer introductiuon and spec itself.  @plehegar is it ok to swap those?

Reply:
Just to clarify on this - I looked at first 5 specs here https://www.w3.org/TR/ and all of them has a few sentences abstract in the beginning of the document before `sotd` section. And it makes sense as you want to quickly read what it is about and then decide whether you need to read it further. This change makes our spec different.

Reply:
indeed. The abstract must appear before the SOTD.
[[
There must be a status section that follows the abstract, […]
]]
https://www.w3.org/pubrules/doc/rules/?profile=CR#sotd